scripts/pkg/build_templates/opensearch-dashboards/deb/debian/postinst

```diff
@@ -29,23 +29,32 @@ if command -v systemd-tmpfiles > /dev/null; then
     systemd-tmpfiles --create opensearch-dashboards.conf
 fi
 
-# Messages
-echo "### NOT starting on installation, please execute the following statements to configure opensearch-dashboards service to start automatically using systemd"
-echo " sudo systemctl daemon-reload"
-echo " sudo systemctl enable opensearch-dashboards.service"
-echo "### You can start opensearch-dashboards service by executing"
-echo " sudo systemctl start opensearch-dashboards.service"
+# Restart the service after an upgrade
+if [ "$1" == "configure" ] && [ "$2" != "" ]; then
```

Comment:
Can we change this line to:
`if [ "$1" = "configure" ] && [ -n "$2" ]; then`

1. `[ foo = bar ]` is POSIX correct. `[ foo == bar ]` is a bashism and should be avoided.
2. `-n` True if the length of string is nonzero, so it should be better than just check `[ "$2" != "" ]`.

Reply:
The change to posix compliance and the usage of "-n" looks good to me, thanks

Reply:
Great. @wieso-itzi if you commit with above changes for both opensearch and opensearch-dashboards, I will approve this PR right after.

Thanks.